src/content/docs/apm/apm-ui-pages/monitoring/external-services/external-services-ui.mdx

```diff
@@ -0,0 +1,143 @@
+
+![Screenshot showing the opening map for external services](./images/intro-example.png "Screenshot showing the opening map for external services")
+
+### View transaction details [#drill-down]
+
+On the initial page for external services, each of the rectangles contains vertices that represent the upstream or downstream services. The vertices are connected across services by edges (lines). When you drill into a particular service, the vertices on those drill-down pages become the service endpoints so you can see transaction detail.
+
+The thickness of the lines represents the throughput for the service, and the darkness of the line is time consumed (throughput times duration).
+
+### Map legend [#map-legend]
+
+In the map legend, you have the option to select two types of services:
+
+  * **Services**: These are services you own and have instrumented.
+  * **Uninstrumented externals**: These are uninstrumented services that you may or may not own.
+
+## How performance metric charts work [#understand-data]
+
+On the opening page of external services, you see these performance metric charts: **Response time**, **Throughput**, and **Error rate**. For APM agents, these initial performance charts are populated by metric data, while for OpenTelemetry, the initial values are populated by sampled data. As you drill below the initial page–whether it's OpenTelemetry or APM agents–each child page is populated by sampled data. This means that if you're not seeing the data you expect on pages that show sampled data, you may need to increase your [sampling](/docs/apm/apm-ui-pages/monitoring/external-services/external-services-setup#adjust-sampling).
+
+The performance metric charts always reflect the data from the page you're viewing, but the set of metrics displayed changes as you drill below the opening page. Here's what you need to know about these metrics:
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+        Metric
+      </th>
+      <th>
+        Screen
+      </th>
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+    <tbody>
+      <tr>
+        <td>
+          Response Time
+        </td>
+        <td>
+          All screens
+        </td>
+        <td>
+          The average duration of sampled calls between two services in the initial view, or between transactions in the other views. How well this represents actual system performance depends on the effective sample rate. This metric is displayed on each level as you drill down.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          Throughput
+        </td>
+        <td>
+          Opening screen
+        </td>
+        <td>
+          ??
```

Comment:
Intentional to have ?? here?

Reply:
@barbnewrelic, yes, I'm still waiting for content from the SME!